### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by lneuhaus**
Code owners will automatically get a requiest for review.

### CODE REVIEW

**Review by josselex — CHANGES_REQUESTED** [review]
@the-tompiler
@ranguwud
Please also join

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* lneuhaus josselex
```
Comment (josselex) [inline CODEOWNERS:1]:
```suggestion
* lneuhaus josselex ranguwud the-tompiler
```

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+* lneuhaus josselex ranguwud the-tompiler
```
Comment (lneuhaus) [inline CODEOWNERS:1]:
```suggestion
* @lneuhaus @josselex @ranguwud @the-tompiler
```

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1 @@
+* @lneuhaus @josselex @ranguwud @the-tompiler
```
Comment (lneuhaus) [inline CODEOWNERS:1]:
```suggestion
* @lneuhaus @josselex @ranguwud @The-Tompiler
```

**Review by josselex — APPROVED** [review]

**Pull request merged**